pkg/reconciler/v1alpha1/autoscaling/kpa_scaler.go

```diff
@@ -84,6 +86,46 @@ func (ks *kpaScaler) getAutoscalerConfig() *autoscaler.Config {
 	return ks.autoscalerConfig.DeepCopy()
 }
 
+func scaleBoundInt32(ctx context.Context, kpa *kpa.PodAutoscaler, key string) int32 {
+	logger := logging.FromContext(ctx)
+	if s, ok := kpa.Annotations[key]; ok {
+		i, err := strconv.ParseInt(s, 10, 32)
+		if err != nil || i < 1 {
+			logger.Debugf("value of %s must be an integer greater than 0", key)
+			return 0
+		}
+		return int32(i)
+	}
+	return 0
+}
+
+func getScaleBounds(ctx context.Context, kpa *kpa.PodAutoscaler) (int32, int32) {
```

Comment:
Should this be an accessor on the KPA resource itself?

Reply:
Done